sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceUtils.scala

```diff
@@ -190,4 +192,15 @@ object DataSourceUtils {
     case LegacyBehaviorPolicy.LEGACY => RebaseDateTime.rebaseGregorianToJulianMicros
     case LegacyBehaviorPolicy.CORRECTED => identity[Long]
   }
+
+  private[sql] def checkDuplicateOptions(
```

Comment:
We can remove `private[sql]` since execution package is already private (SPARK-16964)

Reply:
Ah yeah, thanks for reminding, done in cc250b9.